### PR TITLE
add node server file

### PR DESCRIPTION
**Pull request by jaisi04**
setting up node server

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    var filePath = '.' + request.url;
+    if (filePath == './') {
+        filePath = './index.html';
+    }
+
+    var extname = String(path.extname(filePath)).toLowerCase();
+    var mimeTypes = {
+        '.html': 'text/html',
+        '.js': 'text/javascript',
+        '.css': 'text/css',
```
Comment (rajeevuipath) [inline app.js:18]: Please validate if all these are being used, Lets keep which are being used currently

File: app.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+//refs: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Node_server_without_framework
+var http = require('http');
+var fs = require('fs');
+var path = require('path');
+
+http.createServer(function (request, response) {
+    console.log('request ', request.url);
```
Comment (rajeevuipath) [inline app.js:7]: Prefer to use loggers

**Review by rajeevuipath — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (jaisi04) [reply app.js:18]: now added a boilerplate for the app. will update as per the user specification.

File: app.js (hunk 2 above)
Comment (jaisi04) [reply app.js:7]: changes are not required now.

**Review by rajeevuipath — APPROVED** [review]

**Pull request merged**